homeassistant/components/google_cloud/tts.py

```diff
@@ -223,11 +235,12 @@ async def async_get_tts_audio(self, message, language, options=None):
             {
                 vol.Optional(CONF_GENDER, default=self._gender): GENDER_SCHEMA,
                 vol.Optional(CONF_VOICE, default=self._voice): VOICE_SCHEMA,
-                vol.Optional(CONF_ENCODING, default=DEFAULT_ENCODING): SCHEMA_ENCODING,
+                vol.Optional(CONF_ENCODING, default=self._encoding): SCHEMA_ENCODING,
                 vol.Optional(CONF_SPEED, default=self._speed): SPEED_SCHEMA,
-                vol.Optional(CONF_PITCH, default=self._speed): SPEED_SCHEMA,
-                vol.Optional(CONF_GAIN, default=DEFAULT_GAIN): GAIN_SCHEMA,
-                vol.Optional(CONF_PROFILES, default=[]): PROFILES_SCHEMA,
+                vol.Optional(CONF_PITCH, default=self._pitch): SPEED_SCHEMA,
```

Comment:
Should this use the pitch schema instead?

Reply:
Of course, thanks!